File: src/main/java/org/books/entities/Owner.java
package org.books.entities;

import javax.persistence.*;

@Entity
@Table(name = "owner")
public class Owner {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;

    @Column
    private String name;

    @Column(name = "pass")
    private String password;

    @Column
    private String salt;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public String getSalt() {
        return salt;
    }

    public void setSalt(String salt) {
        this.salt = salt;
    }
}

File: src/main/java/org/books/entities/Owner.kt
package org.books.entities

import javax.persistence.*

@Entity
@Table(name = "owner")
class Owner {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Int = 0

    @Column
    var name: String? = null

    @Column(name = "pass")
    var password: String? = null

    @Column
    var salt: String? = null
}
